.github/workflows/stale.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     recent activity. It will be closed if no further activity occurs. Thank you
     for your contributions."
           days-before-stale: -1
+          days-before-close: -1
```

Comment:
it looks like this will disable [stalebot from marking any issues as stale](https://github.com/actions/stale#days-before-stale), is that the goal?

Reply:
Yes. I think we only want to mark **PRs** as stale.